app/components/Home.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 
 class Home extends Component {
   render() {
-    return <div> Welcome to CLIMBAR</div>;
+    return <div>Connect Collaborate &amp; Create Amazing Climbing Routes</div>;
```

Comment:
We probably want to put this line in the middle of the page and increase the text size :D

Reply:
Done